Source/Core/VideoBackends/D3D/D3DBase.cpp

```diff
@@ -324,45 +293,34 @@ HRESULT Create(HWND wnd)
     UpdateActiveConfig();
   }
 
-  DXGI_SWAP_CHAIN_DESC swap_chain_desc = {};
-  swap_chain_desc.BufferCount = 1;
+  DXGI_SWAP_CHAIN_DESC1 swap_chain_desc = {};
+  swap_chain_desc.BufferCount = NUM_SWAPCHAIN_BUFFERS;
   swap_chain_desc.BufferUsage = DXGI_USAGE_RENDER_TARGET_OUTPUT;
-  swap_chain_desc.OutputWindow = wnd;
   swap_chain_desc.SampleDesc.Count = 1;
   swap_chain_desc.SampleDesc.Quality = 0;
-  swap_chain_desc.Windowed =
-      !SConfig::GetInstance().bFullscreen || g_ActiveConfig.bBorderlessFullscreen;
-
-  DXGI_OUTPUT_DESC out_desc = {};
-  output->GetDesc(&out_desc);
-
-  DXGI_MODE_DESC mode_desc = {};
-  mode_desc.Width = out_desc.DesktopCoordinates.right - out_desc.DesktopCoordinates.left;
-  mode_desc.Height = out_desc.DesktopCoordinates.bottom - out_desc.DesktopCoordinates.top;
-  mode_desc.Format = DXGI_FORMAT_R8G8B8A8_UNORM;
-  mode_desc.Scaling = DXGI_MODE_SCALING_UNSPECIFIED;
-  hr = output->FindClosestMatchingMode(&mode_desc, &swap_chain_desc.BufferDesc, nullptr);
-  if (FAILED(hr))
-    MessageBox(wnd, _T("Failed to find a supported video mode"), _T("Dolphin Direct3D 11 backend"),
-               MB_OK | MB_ICONERROR);
+  swap_chain_desc.Format = DXGI_FORMAT_R8G8B8A8_UNORM;
+  swap_chain_desc.Scaling = DXGI_SCALING_STRETCH;
+  swap_chain_desc.SwapEffect = DXGI_SWAP_EFFECT_FLIP_SEQUENTIAL;
+  swap_chain_desc.Width = xres;
+  swap_chain_desc.Height = yres;
 
-  if (swap_chain_desc.Windowed)
-  {
-    // forcing buffer resolution to xres and yres..
-    // this is not a problem as long as we're in windowed mode
-    swap_chain_desc.BufferDesc.Width = xres;
-    swap_chain_desc.BufferDesc.Height = yres;
-  }
+  // By always creating a stereo swapchain we can toggle Quad-Buffered stereoscopy
+  // while the game is running.
+  swap_chain_desc.Stereo = TRUE;
```

Comment:
That's not documented anywhere, we will need to do some performance testing if we want to know that. But logically speaking it should only result in the allocation of a larger backbuffer.